src/Microsoft.ML.Data/Transforms/NormalizeColumn.cs

```diff
@@ -13,22 +13,21 @@
 using Microsoft.ML.Runtime.Model.Pfa;
 using Microsoft.ML.Runtime.Internal.Internallearn;
 using Newtonsoft.Json.Linq;
+using Microsoft.ML.Runtime.EntryPoints;
+using System.Linq;
```

Comment:
I like how we start with system namespaces, then we get into Microsoft namespaces, then towards the end we go back to system. It sort of reminds me of one of those stories where the protagonist starts at home, goes off to the big city, then comes right back where they started after many adventures, but everything's a bit different. So please keep this.